graphql-dgs-example-java/build.gradle.kts

```diff
@@ -18,7 +18,8 @@ dependencies {
     implementation(project(":graphql-dgs-example-shared"))
     implementation(project(":graphql-dgs-spring-boot-starter"))
     implementation(project(":graphql-dgs-pagination"))
-    implementation(project(":graphql-dgs-subscriptions-websockets-autoconfigure"))
+//    implementation(project(":graphql-dgs-subscriptions-websockets-autoconfigure"))
```

Comment:
remove commented dep?

Reply:
yep, that shouldn't have been committed. Rolled back.